package.json

```diff
@@ -466,6 +466,28 @@
         "contents": "You can create a new Java project by clicking the below button.\n[Create new Java project](command:java.project.create)",
         "when": "!java:projectManagerActivated || java:serverMode != Standard"
       }
+    ],
+    "taskDefinitions": [
+      {
+        "type": "java",
+        "properties": {
+          "elements": {
```

Comment:
A reference about how to support intellisense for constant literals when configuring elements.

https://github.com/microsoft/vscode-java-debug/blob/master/package.json#L346-L373

![constant](https://user-images.githubusercontent.com/14052197/98335989-fd9f3f80-2040-11eb-851a-9b4955fef1b7.gif)

Reply:
@testforstephen beautiful!